api/src/main/java/io/strimzi/api/kafka/model/KafkaClusterSpec.java

```diff
@@ -211,15 +212,26 @@ public void setJvmOptions(JvmOptions jvmOptions) {
         this.jvmOptions = jvmOptions;
     }
 
+    @JsonInclude(JsonInclude.Include.NON_EMPTY)
+    @Description("JMX Options for Kafka brokers")
+    @JsonProperty("jmxRemote")
+    public KafkaJmxRemote getJmxRemote() {
+        return jmxRemote;
+    }
+
+    public void setJmxRemote(KafkaJmxRemote kafkaJmxRemote) {
+        this.jmxRemote = kafkaJmxRemote;
+    }
+
     @JsonInclude(JsonInclude.Include.NON_EMPTY)
     @Description("The Prometheus JMX Exporter configuration. " +
             "See https://github.com/prometheus/jmx_exporter for details of the structure of this configuration.")
-    public Map<String, Object> getMetrics() {
-        return metrics;
+    public Map<String, Object> getPrometheusMetrics() {
```

Comment:
I think this renames the existing `metrics` field which would be an API change and we cannot do this so easily. So I think that despite the nme being a bit confusing with the JMX stuff added, we should keep the original name.

Reply:
Ok no worries will do so!

Reply:
@scholzj I was wondering if I could get your opinion on keeping `$KAFKA_PROMETHEUS_METRICS_ENABLED` rather than `$KAFKA_METRICS_ENABLED`.  The reason being that I think that someone looking at the `kafka_run.sh` will get confused about why there is a `JMX_REMOTE` and `KAFKA_METRICS_ENABLED`. This won't affect anything in the CRD but make it clearer about what is happening in the `kafka_run.sh` for people working on the script.

Reply:
I would stay consistent, avoiding different naming (where possible) between API and env vars in scripts. Of course, it's just my point of view :-)